conda_smithy/configure_feedstock.py

```diff
@@ -402,11 +411,15 @@ def render_appveyor(jinja_env, forge_config, forge_dir):
     for platform, arch in [['win-32', 'x86'], ['win-64', 'x64']]:
         with fudge_subdir(platform, build_config=meta_config(meta)):
             meta.parse_again()
-            matrix = compute_build_matrix(
-                meta,
-                forge_config.get('matrix'),
-                forge_config.get('channels', {}).get('sources', tuple())
-            )
+            if meta.noarch:
+                # do not build noarch, include noarch: python packages on AppVeyor.
+                matrix = ()
```

Comment:
This is typically a `list`. I don't think there is any harm with it being a `tuple`, but we should be consistent here.